.gitignore

```diff
@@ -11,7 +11,6 @@
 *.rej
 *.exe
 .*.swp
-workflows
```

Comment:
This was meant to ignore the `workflows` binary, which is still valuable in this repository.

I suggest to change it to `/workflows` which will still achieve the same and won't ignore `any-other-dir/workflows/...`

Reply:
ah I just thought the binary exclusion would be a shortcut (since I'm so used to the gui repo target folder for the build output), but yeah makes a lot of sense! Should now be as suggested 👌